apartmenttherapy/order-service
# Elaine/close listing

#### What does this PR do & why?
Adding functionality to have the order service close the order listing when the customer pays for it.

#### How can this be manually tested?
- Tests should run.
- Paying for an order should automatically close the listing.

#### Any questions or concerns about this change?
*(Any significant changes you're introducing? What side effects might this have? What other parts of the app could this affect?)*


#### Deployment Notes
*(e.g. Does this require a migration or special infrastructure change? Clearing the cache? Third party service?)*


#### Link to story card and other documentation
https://app.asana.com/0/694906336753887/1113441749015952

#### GIF


#### Have you remembered to...?
